lib/main.coffee

```diff
@@ -6,6 +6,10 @@ module.exports =
       default: ''
       description: 'Enter the path to your phpcs executable.'
       order: 1
+    disableWhenNoConfigFile:
+      type: 'boolean'
+      default: false
+      description: 'Disable linter when not found the configuration file (`phpcs.xml` or `phpcs.ruleset.xml`)'
```

Comment:
Also specify an order or it will be shown at the bottom. (This should probably show right below codeStandardOrConfigFile)